documentation/src/docs/asciidoc/docinfos/index-docinfo.html

```diff
@@ -0,0 +1 @@
+<link rel="icon" type="image/png" href="{{ site.baseurl }}/img/junit5-logo.png">
```

Comment:
strange, it replaced it when I did a publish. It created the gh-pages folder and when I opened the index.html files for both the javadoc and the userguide it had the favicon.